app/build.gradle

```diff
@@ -78,6 +78,7 @@ dependencies {
 
     implementation 'com.squareup.retrofit2:retrofit:2.9.0'
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
+    implementation "com.squareup.okhttp3:logging-interceptor:4.9.3"
```

Comment:
I added a logger in the network layer. I used this to debug the calls.

Reply:
Classic!